extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1390,4 +1391,15 @@ export class AmpStoryPage extends AMP.BaseElement {
   isAd() {
     return this.element.hasAttribute(ADVERTISEMENT_ATTR_NAME);
   }
+
+  /**
+   * Sets text styles for descendants of the
+   * <amp-story-page> element.
+   * @private
+   */
+  setDescendentCssTextStyles_() {
+    this.mutateElement(() => {
```

Comment:
Nit: the `buildCallback` already runs in a mutate phase, so it's the one exception where you don't need to wrap your DOM mutations in a `mutateElement()` statement